Gemfile

```diff
@@ -1,13 +1,15 @@
 # frozen_string_literal: true
 
-ruby '3.3.1'
+ruby '3.3.6'
 
 source 'https://rubygems.org'
 
 gem 'avro', '~> 1.10.0'
 gem 'avro-resolution_canonical_form', '>= 0.2.0'
 gem 'bootsnap', require: false
+gem 'drb' # TODO: Remove with rails 7.1
 gem 'grape'
+gem 'mutex_m' # TODO: Remove with rails 7.1
```

Comment:
These are to address warnings like: `mutex_m.rb was loaded from the standard library, but will no longer be part of the default gems starting from Ruby 3.4.0.`

Reply:
Does Rails 7.1 explicitly include these, or they're no longer used in Rails 7.1?

Reply:
They are bundled with Rails 7.1: https://github.com/rails/rails/pull/48907